storage/chunkDataPacks.go

```diff
@@ -10,6 +10,8 @@ type ChunkDataPacks interface {
 	// Store inserts the chunk header, keyed by chunk ID.
 	Store(c *flow.ChunkDataPack) error
```

Comment:
does it still make sense to have this function? could we just remove it? and then rename StoreMul to Store